modules/http/src/main/java/org/mule/module/http/internal/request/MuleEventToHttpRequest.java

```diff
@@ -74,7 +74,10 @@ public HttpRequestBuilder create(MuleEvent event, String resolvedMethod, String
 
         for (String outboundProperty : event.getMessage().getOutboundPropertyNames())
         {
-            builder.addHeader(outboundProperty, event.getMessage().getOutboundProperty(outboundProperty).toString());
+            if (!builder.getHeaders().containsKey(outboundProperty))
```

Comment:
Wouldn't this make the 3.6.x branch different from 3.x? Outbound properties take precedence over the request builder AFAIK. Why is this needed?